example/lib/src/navigation.dart

```diff
@@ -44,6 +44,85 @@ class NavigationDemo extends StatelessWidget {
   }
 }
 
+class TabsDemo extends StatefulWidget {
```

Comment:
It looks like this could be a StatelessWidget?

Reply:
Yes!